com.github.tmewett.BrogueCE.metainfo.xml

```diff
@@ -21,9 +22,18 @@
     <url type="bugtracker">https://github.com/tmewett/BrogueCE/issues</url>
     <screenshots>
       <screenshot type="default">
-        <image>https://dl.flathub.org/repo/screenshots/com.github.tmewett.BrogueCE-stable/752x423/com.github.tmewett.BrogueCE-7a9363eb8a1ca6ede2f70bf9a1f6a339.png</image>
+        <image>https://dl.flathub.org/repo/screenshots/com.github.tmewett.BrogueCE-stable/752x423/com.github.tmewett.BrogueCE-5d4557f183aa165a38963c3dfde999a2.png</image>
         <caption>Main Window</caption>
       </screenshot>
+<!--
```

Comment:
Is commenting this intentional?

Reply:
Yes, the screenshots are not merged yet so **buildbot** will fail and complain that the URL is broken.
Also I'm not sure if it's the best way to link the images on Flathub but it should be less fragile.